index.html

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <section class="main">
+        <container class="main-grid">
+            
+            <!--------------------------------LEFT COLUMN 1-------------------------------->
+            <div class="left-side-icons">
+                <div class="left-icons-wrapper">
+                    <img id="skull" src="/Icons/skull.png"> 
+                    <img id="idea" src="/Icons/idea.png">
+                    <img id="rocket" src="/Icons/rocket.png">
+                    <img id="connections" src="/Icons/connections.png">
+                    <img id="spider" src="/Icons/spider.png">
+                    <img id="sand-castle" src="/Icons/sand-castle.png">
+                    <img id="jellyfish" src="/Icons/jellyfish.png">
+                    <img id="crab" src="/Icons/crab.png">
+                    <img id="wasp" src="/Icons/wasp.png">
+                </div>
+            </div>    
+            <!--------------------------------LEFT COLUMN 2-------------------------------->
+            <div class="left-menu">
+                <div class="left-menu-wrapper">
+                    <div class="left-menu-item">
+                        
```

Comment:
unnecessary space

Reply:
Thanks Phil.  I think I had left them in there to remind me to put something in there, or it's where I'd taken something out, but either way, noted that they should not be there.